Automattic/wp-calypso
# Bluehost on Plans: Feature refresh for Bluehost launch

Related to #

## Proposed Changes

* Adds the feature enhancements done for Bluehost based landing pages to Calypso
* Document: https://docs.google.com/document/d/1gqQWVtccPckyOsJv9rLQ9yvIRYOezWQSoQzBHVJG5Rg/edit

<img width="845" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/9744cfb0-6b97-4e02-b0bb-55f6fc21580b">


## Testing Instructions



* Go to  `/start`
* Reach plans step
* Make sure all the suggested feature changes and tooltip changes are visible

## Pre-merge Checklist
